.gitignore

```diff
@@ -0,0 +1,12 @@
+# Node build artifacts
+node_modules/
+npm-debug.log
+
+# Local development
+*.env
+*.dev
+.DS_Store
+
+# Docker
```

Comment:
Does your application use Docker?

Reply:
No but not sure to use it so still leaving it.
But I will delete this.